### PR TITLE
1.20.1

### PR DESCRIPTION
**Pull request by Arborsm**
#2

### CODE REVIEW

File: src/main/java/com/ccr4ft3r/geotaggedscreenshots/util/RenderUtil.java [hunk 1]
```diff
@@ -1,18 +1,39 @@
...
+        RenderSystem.setShader(GameRenderer::getPositionTexShader);
+        BufferBuilder bufferbuilder = Tesselator.getInstance().getBuilder();
+        bufferbuilder.begin(VertexFormat.Mode.QUADS, DefaultVertexFormat.POSITION_TEX);
+        Matrix4f matrix4f = guiGraphics.pose().last().pose();
+        bufferbuilder.vertex(matrix4f, (float)p_93114_, (float)p_93117_, (float)p_93118_).uv(p_93119_, p_93122_).endVertex();
+        bufferbuilder.vertex(matrix4f, (float)p_93115_, (float)p_93117_, (float)p_93118_).uv(p_93120_, p_93122_).endVertex();
+        bufferbuilder.vertex(matrix4f, (float)p_93115_, (float)p_93116_, (float)p_93118_).uv(p_93120_, p_93121_).endVertex();
+        bufferbuilder.vertex(matrix4f, (float)p_93114_, (float)p_93116_, (float)p_93118_).uv(p_93119_, p_93121_).endVertex();
+        BufferUploader.drawWithShader(bufferbuilder.end());
+    }
```
Comment (CCr4ft3r) [inline src/main/java/com/ccr4ft3r/geotaggedscreenshots/util/RenderUtil.java]: This seems like a direct copy of GuiComponent#blit (judging by the obfuscated names)?
Please update the variable and parameter names.
All of these 3 new methods should be private static.

You can use this example as a guide: https://github.com/LGatodu47/ScreenshotViewer/blob/08116944b6bce9077b705b3904c012dd98ba4b32/src/main/java/io/github/lgatodu47/screenshot_viewer/screens/ScreenshotWidget.java#L142

File: src/main/java/com/ccr4ft3r/geotaggedscreenshots/util/RenderUtil.java [hunk 2]
```diff
@@ -1,18 +1,39 @@
...
-        GuiComponent.blit(matrixStack, xOffset, yOffset, scaledWidth, scaledHeight, 0, 0, width, height, width, height);
+        blit(guiGraphics, xOffset, yOffset, scaledWidth, scaledHeight, 0, 0, width, height, width, height);
         RenderSystem.disableBlend();
     }
+    public static void blit(GuiGraphics guiGraphics, int p_283671_, int p_282377_, int p_282058_, int p_281939_, float p_282285_, float p_283199_, int p_282186_, int p_282322_, int p_282481_, int p_281887_) {
+        blit(guiGraphics, p_283671_, p_283671_ + p_282058_, p_282377_, p_282377_ + p_281939_, 0, p_282186_, p_282322_, p_282285_, p_283199_, p_282481_, p_281887_);
+    }
+
+    static void blit(GuiGraphics guiGraphics, int p_282732_, int p_283541_, int p_281760_, int p_283298_, int p_283429_, int p_282193_, int p_281980_, float p_282660_, float p_281522_, int p_282315_, int p_281436_) {
+        innerBlit(guiGraphics, p_282732_, p_283541_, p_281760_, p_283298_, p_283429_, (p_282660_ + 0.0F) / (float)p_282315_, (p_282660_ + (float)p_282193_) / (float)p_282315_, (p_281522_ + 0.0F) / (float)p_281436_, (p_281522_ + (float)p_281980_) / (float)p_281436_);
```
Comment (CCr4ft3r) [inline src/main/java/com/ccr4ft3r/geotaggedscreenshots/util/RenderUtil.java]: What's the intention of adding zero to int? `p_281522_ + 0.0F`
The second cast `(float)p_281436_` should be sufficient (for an int division that should result in a float only one operand needs to be of the type float)

**Review by CCr4ft3r — CHANGES_REQUESTED** [review]
Only two minor remarks.

File: src/main/java/com/ccr4ft3r/geotaggedscreenshots/util/RenderUtil.java (hunk 2 above)
Comment (Arborsm) [reply src/main/java/com/ccr4ft3r/geotaggedscreenshots/util/RenderUtil.java]: I just direct copied those things haha,gonna change it now

**Comment by Arborsm** [comment]
ok fix it

**Review by CCr4ft3r — APPROVED** [review]
lgtm :+1: Thx for your contribution. I appreciate it :)

**Pull request merged**